core/log_adapter.go

```diff
@@ -0,0 +1,37 @@
+package core
+
+import (
+	tmlog "github.com/celestiaorg/celestia-core/libs/log"
+	"go.uber.org/zap"
+
+	logging "github.com/ipfs/go-log/v2"
+)
+
+var log = logging.Logger("core")
+
+// logAdapter is needed to push through our logger to celestia-core
```

Comment:
Can you explain a bit more what this is doing?

Reply:
And is it really necessary to import another dependency for it?

Reply:
Or maybe we could consider using a different logger then.

Reply:
> Or maybe we could consider using a different logger then.

That won't help

Reply:
> And is it really necessary to import another dependency for it?

We were using/importing Zap indirectly before, as go-log is just a useful wrapper for it. Now we import it directly, so nothing really changed.

Reply:
> Can you explain a bit more what this is doing?

Tendermint has its own logger interface. The adapter implements it over Zap, so we can see all the logs in one unified logger. Even there are 3 different loggers mentions here, all of them use one and only Zap

Reply:
If we already have the tendermint logger [interface](https://github.com/tendermint/tendermint/blob/db690c3b68e31568c2f3832f85960ecb39dc7817/libs/log/logger.go#L32) as a dep why don't we just use the default logger [implementation](https://github.com/tendermint/tendermint/blob/db690c3b68e31568c2f3832f85960ecb39dc7817/libs/log/default.go#L29) of the same package?

Reply:
Because having two loggers writing in different formats in the same stdout is not a good idea. If you are talking about choosing one logger and the one from tendermint, instead of go-log, then we are going back to https://github.com/celestiaorg/celestia-node/issues/39. Pls, continue the discussion there. TO close it, we need your opinion at last.

Reply:
>  If you are talking about choosing one logger and the one from tendermint, instead of go-log, then we are going back to #39. 

#39 seems to be about the question if there should be a (package) global var for a logger or if the logger should be passed in per instance. I'm generally against global variables unless there are really, really compelling reasons for them. 

Yes, let's use the tendermint logger for now and close that issue.